tests/units.ua

```diff
@@ -709,7 +708,9 @@ B ← |1 (⨬(+⊃(B-1|B-2)|1)<2.)
 ⍤⤙≍ ⋕ ⍜csv∘ . ↯3_4⇡12
 ⍤⤙≍ ∵⍣⋕∘ ⍜csv∘ . [{"Foo" "Bar"} [1 2] [3 4] [5 6]]
 ⍤⤙≍ [{"1" "2" ""} {"3" "" ""} {"4" "5" "6"}] °csv "1,2\n3\n4,5,6\n"
+⍤⤙≍ [{"1" "2" ""} {"3" "" ""} {"4" "5" "6"}] ⬚@;°csv "1;2\n3\n4;5;6\n"
```

Comment:
I don't think this behavior should be for `fill(..) un csv`.
It should be for `un fill(..) csv`.
I'll use this PR as a starting point to enabled that.

Reply:
I don't think I'm knowledgable enough to implement this in any measure.